app/src/main/res/values/strings.xml

```diff
@@ -1162,7 +1162,7 @@
     <string name="rt_rethink_dns">RethinkDNS</string>
 
     <string name="rt_dialog_title">Save Changes</string>
-    <string name="rt_dialog_message">Do you want to apply newly configured blocklists?</string>
+    <string name="rt_dialog_message">Apply newly configured blocklists?</string>
     <string name="rt_dialog_positive">Apply</string>
```

Comment:
I would go with all "OK" for positive actions, (and it is a mix now).
Putting the function of actions on buttons give more modern style-guide points.

Reply:
Most of the action buttons have text that directly relate to either the title of the dialog or its content. And so, blanket changing all of it to `OK` wouldn't come off as nicely.